mizar/obj/droplet.py

```diff
@@ -37,7 +38,7 @@ def __init__(self, name, obj_api, opr_store, spec=None):
         self.store = opr_store
         self.ip = ""
         self.mac = ""
-        self.phy_itf = 'eth0'
+        self.phy_itf = netifaces.interfaces()[1]
```

Comment:
Same as above comment, second interface is not always the default network interface.